pycbc/psd/variation.py

```diff
@@ -0,0 +1,118 @@
+""" PSD Variation """
+
+import numpy
+import logging
+
+from pycbc.types import TimeSeries, zeros
+
+def calc_psd_variation(strain, psd_short_segment, psd_long_segment, 
+                       overlap, low_freq, high_freq):
+    """Calculates time series of PSD variability
+
+    This function first splits the segment up in to 512 second chunks. It 
+    then calculates the PSD over this 512 second period as well as in 4 
+    second chunks throughout each 512 second period. Next the function 
+     estimates how different the 4 second PSD is to the 512 second PSD and 
```

Comment:
Spacing still looks off, which is what I think the documentation build is complaining about.